### PR TITLE
Redfish: Implement LocationIndicatorActive Property for Manager

### PR DESCRIPTION
**Pull request by ChicagoDuan**
Implement LocationIndicatorActive property for Manager.
We can use the following command to set LocationIndicatorActive to true or false:
curl -k -H "X-Auth-Token: $token" -X PATCH -d '{"LocationIndicatorActive":false}' https://${bmc}/redfish/v1/Managers/bmc

Signed-off-by: Chicago Duan <duanzhijia01@inspur.com>

### CODE REVIEW

**Review by santoshpuranik — CHANGES_REQUESTED** [review]

File: redfish-core/lib/managers.hpp [hunk 1]
```diff
@@ -37,6 +38,52 @@
...
+                BMCWEB_LOG_DEBUG << "DBUS response error: " << ec;
+                messages::internalError(aResp->res);
+                return;
+            }
+
+            for (const auto& [objectPath, serviceMap] : subtree)
+            {
+                sdbusplus::message::object_path path(objectPath);
+                std::string managerName = path.filename();
+                if (managerName != "bmc")
```
[the remark this answers is not in the document]
Comment (ChicagoDuan) [reply redfish-core/lib/managers.hpp:69]: I considered the situation of multiple Managers. If there is no such situation, I will modify it.

[the remark this answers is not in the document]
Comment (ChicagoDuan) [reply redfish-core/lib/managers.hpp:69]: > Why was this check needed? Are there cases where we have multiple BMCs? Even if we do, wouldn't the LED associations take care of getting to the right LED?

I considered the situation of multiple Managers. If there is no such situation, I will modify it.

**Comment by gtmills** [comment]
jenkins run tests please

**Review by jinuthomas — APPROVED** [review]

**Review by jinuthomas — APPROVED** [review]
looks good to me , will wait for the others to also go through this

**Review by lxwinspur — APPROVED** [review]

**Comment by ChicagoDuan** [comment]
> @ChicagoDuan : Could you send this upstream too? I see https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/43179, but that implementation is quite different

Sure

**Review by gtmills — APPROVED** [review]
Going to default some to Santosh / Jinu here. Looks fine.

**Comment by ChicagoDuan** [comment]
> @ChicagoDuan : Could you send this upstream too? I see https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/43179, but that implementation is quite different

Done : https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/50494

**Pull request merged**